pkg/operation/shoot/types.go

```diff
@@ -81,6 +81,7 @@ type Shoot struct {
 	NodeLocalDNSEnabled        bool
 	Networks                   *Networks
 	ExposureClass              *gardencorev1alpha1.ExposureClass
+	BackupEntryName            string
```

Comment:
Should this variable be a pointer? 
I am just thinking if it can be used somewhere without be properly initialized and this time it will be `""` instead of `"--"`.

Reply:
Other similar fields are not pointers. From my perspective, it would only make sense to make it a pointer if a `nil` would be a valid value (as with `ExternalClusterDomain`), but this is not the case. As for "it can be used somewhere without be properly initialized" it is now initialized at the earliest possible place, and in the same place most of the variables in this struct are initialized. I don't think this means we should make all of them pointers too.